runelite-client/src/main/java/net/runelite/client/plugins/kittennotifier/KittenNotifierConfig.java

```diff
@@ -0,0 +1,20 @@
+package net.runelite.client.plugins.kittennotifier;
```

Comment:
This config file is useless since the plugin is very explicit in what it does. And the following config is the same thing as the plugin being on.

Reply:
The plugin tells if your kitten has grown into a cat, the config adds the functionality of alerting on kitten needs or attention. Players could just add these lines into chat notifications

Reply:
Then why does it exist in the first place